src/errors/Messages.js

```diff
@@ -105,6 +107,10 @@ const Messages = {
 
   VANITY_URL: 'This guild does not have the VANITY_URL feature enabled.',
 
+  INVITE_RESOLVE_CODE: `Couldn't resolve the code to fetch the invite.`,
+
+  UNKNOWN_INVITE: `Unknown invite.`,
```

Comment:
I'm not sure about using errors this way, this type of error is usually an error returned by the Discord API but no endpoint was created to fetch a unique invite so I took the freedom to create an error if the invitation code did not exist.

Reply:
I would suggest changing `UNKNOWN_INVITE` to `INVITE_NOT_FOUND` or something. Having exactly same key as an `APIError` isn't a good idea. Also, while you're on it, can you add some more content to the error message that provides a little more info about the error?

Reply:
I think that it will be a nice change 👍